scripts/check-whitespace.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "Files with trailing whitespace:"
+# From /
+
+ws="$(echo -ne ' \t\v')"
+
+count=0
+for file in $(git grep -IE "[$ws]$" -- $* | cut -d: -f1 | sort -u); do
```

Comment:
I would change this to be override-able:
```bash
FILES ?= $(git grep...)
for file in $FILES; do
```
Then we could add this to `make code_checks` with `FILES=$(CHANGED)`

Reply:
i think we already do that in circle (check for whitespaces). i committed this because it happens a lot to me and one of these days i'm going to loose the script :)
feel free to modify it